<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Aero Control Center</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>780</width>
      <height>551</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="mainTab">
     <attribute name="title">
      <string>Main Tab</string>
     </attribute>
     <widget class="QPushButton" name="fanGamingBtn">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>400</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Gaming</string>
      </property>
     </widget>
     <widget class="QPushButton" name="fanNormalBtn">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>400</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Normal</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="cpuView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
     <widget class="QGraphicsView" name="gpuView">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>20</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
     <widget class="QGraphicsView" name="memoryView">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>20</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
     <widget class="QLabel" name="fanModeLabel">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>370</y>
        <width>211</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Fan Modes</string>
      </property>
     </widget>
     <widget class="QPushButton" name="fanQuietBtn">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>440</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Quiet</string>
      </property>
     </widget>
     <widget class="QPushButton" name="fanCustomBtn">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>440</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Custom</string>
      </property>
     </widget>
     <widget class="QSlider" name="fanCustomSlider">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>470</y>
        <width>171</width>
        <height>26</height>
       </rect>
      </property>
      <property name="minimum">
       <number>25</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="singleStep">
       <number>5</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBothSides</enum>
      </property>
      <property name="tickInterval">
       <number>5</number>
      </property>
     </widget>
     <widget class="QLabel" name="fanCustomPercent">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>470</y>
        <width>51</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>0%</string>
      </property>
     </widget>
     <widget class="QPushButton" name="setFanSpeedBtn">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>470</y>
        <width>80</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Set</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="fan2RPM">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>440</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="fan1RPM">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>380</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="fan1Label">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>380</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Source Code Pro</family>
        <pointsize>15</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Fan 1</string>
      </property>
     </widget>
     <widget class="QLabel" name="fan2Label">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>440</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Source Code Pro</family>
        <pointsize>15</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Fan 2</string>
      </property>
     </widget>
     <widget class="QLabel" name="cpuLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>220</y>
        <width>256</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>CPU: </string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="rgbTab">
     <attribute name="title">
      <string>RGB Fusion</string>
     </attribute>
     <widget class="QComboBox" name="rgbModes">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>440</y>
        <width>141</width>
        <height>26</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Static</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Breathing</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Wave</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Fade on Keypress</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Marquee</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Flash on Keypress</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Neon</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Rainbow Marquee</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Raindrop</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Circle Marquee</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Hedge</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Rotate</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Custom 1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Custom 2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Custom 3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Custom 4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Custom 5</string>
       </property>
      </item>
     </widget>
     <widget class="QCheckBox" name="rgbEnabled">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>440</y>
        <width>84</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Enabled</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="rgbRandom">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>470</y>
        <width>84</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Random</string>
      </property>
     </widget>
     <widget class="QSlider" name="rgbBrightnessSlider">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>440</y>
        <width>161</width>
        <height>24</height>
       </rect>
      </property>
      <property name="maximum">
       <number>2</number>
      </property>
      <property name="pageStep">
       <number>2</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBothSides</enum>
      </property>
     </widget>
     <widget class="QLabel" name="rgbBrightnessLabel">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>440</y>
        <width>71</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Brightness</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="rgbPreview">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>751</width>
        <height>421</height>
       </rect>
      </property>
     </widget>
     <widget class="QSlider" name="rgbSpeedSlider">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>470</y>
        <width>160</width>
        <height>24</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
      <property name="pageStep">
       <number>2</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBothSides</enum>
      </property>
      <property name="tickInterval">
       <number>1</number>
      </property>
     </widget>
     <widget class="QLabel" name="rgbSpeedLabel">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>470</y>
        <width>57</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Speed</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="rgbRedBox">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>440</y>
        <width>42</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="rgbGreenBox">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>440</y>
        <width>42</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="rgbBlueBox">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>440</y>
        <width>42</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>440</y>
        <width>31</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>RGB</string>
      </property>
     </widget>
     <widget class="QPushButton" name="rgbSetBtn">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>470</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Set</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>fanCustomBtn</sender>
   <signal>clicked()</signal>
   <receiver>fanCustomSlider</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>494</y>
    </hint>
    <hint type="destinationlabel">
     <x>135</x>
     <y>529</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fanGamingBtn</sender>
   <signal>clicked()</signal>
   <receiver>fanCustomSlider</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>454</y>
    </hint>
    <hint type="destinationlabel">
     <x>135</x>
     <y>529</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fanNormalBtn</sender>
   <signal>clicked()</signal>
   <receiver>fanCustomSlider</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>454</y>
    </hint>
    <hint type="destinationlabel">
     <x>135</x>
     <y>529</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fanQuietBtn</sender>
   <signal>clicked()</signal>
   <receiver>fanCustomSlider</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>494</y>
    </hint>
    <hint type="destinationlabel">
     <x>135</x>
     <y>529</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fanQuietBtn</sender>
   <signal>clicked()</signal>
   <receiver>fanCustomPercent</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>495</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>531</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fanNormalBtn</sender>
   <signal>clicked()</signal>
   <receiver>fanCustomPercent</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>455</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>531</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fanGamingBtn</sender>
   <signal>clicked()</signal>
   <receiver>fanCustomPercent</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>455</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>531</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fanCustomBtn</sender>
   <signal>clicked()</signal>
   <receiver>fanCustomPercent</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>495</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>531</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fanQuietBtn</sender>
   <signal>clicked()</signal>
   <receiver>setFanSpeedBtn</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>495</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>525</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fanNormalBtn</sender>
   <signal>clicked()</signal>
   <receiver>setFanSpeedBtn</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>455</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>525</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fanGamingBtn</sender>
   <signal>clicked()</signal>
   <receiver>setFanSpeedBtn</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>455</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>525</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fanCustomBtn</sender>
   <signal>clicked()</signal>
   <receiver>setFanSpeedBtn</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>495</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>525</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
